src/Microsoft.TestPlatform.ObjectModel/Client/Events/DiscoveryCompleteEventArgs.cs

```diff
@@ -27,11 +27,6 @@ public class DiscoveryCompleteEventArgs : EventArgs
         IList<string> partiallyDiscoveredSources,
         IList<string> notDiscoveredSources)
     {
-        // This event is always raised from the client side, while the total count of tests is maintained
```

Comment:
I had to remove this comment because we are now also using this event args inside the parallel discovery.